### PR TITLE
wpg: escape column names in schema

### LINKED ISSUE #225: detect invalid use of "from" and "to" in config
You can't create a PG column named "from" or "to" and so we should detect that in the config file and return a helpful error message.

### PR DESCRIPTION
**Pull request by worm-emoji**
this allows for more permissive column names such as `from` or
other reserved pg keywords.

resolves #225

### CODE REVIEW

File: wpg/pg.go [hunk 1]
```diff
@@ -41,6 +41,10 @@ type Column struct {
 	Type string `db:"data_type"json:"type"`
 }
 
+func (c Column) String() string {
```
Comment (worm-emoji) [inline wpg/pg.go:44]: maybe overkill to add this function, happy to replace it with just doing the escaping inline

**Comment by worm-emoji** [comment]
we could also go much further here and include indexes, tables, and other named parts of schema

File: wpg/pg.go [hunk 2]
```diff
@@ -41,6 +41,10 @@ type Column struct {
 	Type string `db:"data_type"json:"type"`
 }
 
+func escaped(s string) string {
```
Comment (worm-emoji) [inline wpg/pg.go:44]: i guess some optimization _could_ be checking if the column name == a [postgres reserved keyword](https://www.postgresql.org/docs/current/sql-keywords-appendix.html) but there are so many of those. plus would need to be maintained as postgres adds keywords. `"` should ~always work.

Comment (ryandotsmith) [reply wpg/pg.go:45]: Hrm. It's kind of annoying to have everything quoted. I wonder if we can build the set of reserved words on startup using this query:

```sql
select word from pg_get_keywords() where catdesc = 'reserved';
```

Comment (worm-emoji) [reply wpg/pg.go:45]: i thought about that, but doesn't that impose limitations such as requiring a pg connection to run shovel with the `-schema` flag

Comment (worm-emoji) [reply wpg/pg.go:45]: imo i think if we check if it's in the reserved word list, best to just bake it into the code and update as needed

Comment (ryandotsmith) [reply wpg/pg.go:45]: I'm fine with that too. Perhaps we can use `go generate` to create the list. I don't think this will change too much.

**Pull request merged**